pagination-practice/tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
+  theme: {
+    extend: {},
```

Comment:
can you use some basic configuration for tailwind like the theme primary secondary colors, font sizes and also import a font from google.